e2e/testsuite/testconfig.go

```diff
@@ -62,7 +62,7 @@ const (
 	// TODO: https://github.com/cosmos/ibc-go/issues/4965
 	defaultHyperspaceTag = "20231122v39"
 	// defaultHermesTag is the tag that will be used if no relayer tag is specified for hermes.
-	defaultHermesTag = "luca_joss-channel-upgrade-authority"
+	defaultHermesTag = "sean-channel-upgradability"
```

Comment:
have we been using this tag for the last few months?

Reply:
We were using `luca_joss-...`, but when these attributes were removed, Luca discovered while debugging that there was a bug in hermes and fixed it in this new tag.

Reply:
For more context, the branch `sean-channel-upgradability` was the branch for the channel upgrade beta implementation and the `luca_joss-channel-upgrade-authority` was the branch adding the changes that came after the beta (e.g. initiating an upgrade only with authority). Between the release of channel upgrade and now the branch `luca_joss-channel-upgrade-authority` has been merged in `sean-channel-upgradability` so that we only have one open PR related to channel upgrade. New Hermes Docker images for channel upgrade, until it is merged, will be in the tag sean-channel-upgradability.